packages/cli/src/commands/integration/remove.ts

```diff
@@ -0,0 +1,357 @@
+import { parseArguments } from '../../util/get-args';
+import handleError from '../../util/handle-error';
+import type { Team } from '@vercel-internals/types';
+import { getFirstConfiguration } from '../../util/integration/fetch-marketplace-integrations';
+import confirm from '../../util/input/confirm';
+import { removeIntegration } from '../../util/integration/remove-integration';
+import {
+  disconnectResourceFromAllProjects,
+  disconnectResourceFromProject,
+} from '../../util/integration/disconnect-resource-from-project';
+import { deleteResource } from '../../util/integration/delete-resource';
+
+interface RemoveCommandFlags {
+  '--delete'?: boolean | undefined;
+  '--unlink-all'?: boolean | undefined;
+  '--yes'?: boolean | undefined;
+}
+
+export async function remove(client: Client) {
+  let parsedArguments = null;
+  const flagsSpecification = getFlagsSpecification(removeSubcommand.options);
+
+  try {
+    parsedArguments = parseArguments(client.argv.slice(3), flagsSpecification);
+  } catch (error) {
+    handleError(error);
+    return 1;
+  }
+
+  const { team } = await getScope(client);
+  if (!team) {
+    client.output.error('Team not found.');
+    return 1;
+  }
+
+  if (parsedArguments.args.length < 2) {
+    client.output.error(
+      'You must specify a resource or integration. See `--help` for details.'
+    );
+    return 1;
+  }
+
+  if (parsedArguments.args.length > 3) {
+    client.output.error(
+      'Cannot specify more than one project at a time. Use `--unlink-all` to unlink the specified resource from all projects.'
+    );
+    return 1;
+  }
+
+  if (
+    parsedArguments.args.length === 3 &&
+    parsedArguments.flags['--unlink-all']
+  ) {
+    client.output.error(
+      'Cannot specify a project while using the `--unlink-all` flag.'
+    );
+    return 1;
+  }
+
+  const resourceOrIntegrationName = parsedArguments.args[1];
+  const searchForIntegration =
+    parsedArguments.args.length === 2 &&
+    !parsedArguments.flags['--delete'] &&
+    !parsedArguments.flags['--unlink-all'];
+
+  if (searchForIntegration) {
+    return await handleRemoveIntegration(
+      client,
+      team,
+      resourceOrIntegrationName,
+      !!parsedArguments.flags['--yes']
+    );
+  }
+
+  return await handleUnlinkOrDeleteResource(
+    client,
+    team,
+    resourceOrIntegrationName,
+    parsedArguments.args,
+    parsedArguments.flags
+  );
+}
+
+// $ vc integration remove [integration-name]
+async function handleRemoveIntegration(
+  client: Client,
+  team: Team,
+  integrationName: string,
+  skipConfirmation: boolean
+): Promise<number> {
+  client.output.spinner('Retrieving integration…', 500);
+  const integrationConfiguration = await getFirstConfiguration(
+    client,
+    integrationName
+  );
+
+  if (!integrationConfiguration) {
+    client.output.error(`No integration ${chalk.bold(integrationName)} found.`);
+    return 0;
+  }
+
+  // Confirm removal desired
```

Comment:
suggestion: remove in favor of const for if statement

Reply:
IMO the skip flag check & confirmation function naming is already clear enough on what the intention is.